src/server/services/userHandler.js

```diff
@@ -1,5 +1,10 @@
 const users = [{ id: 1, username: 'annie', password: 'hola' }];
 
+/**
+ * Return an user information by username from registered/active Users database.
```

Comment:
you should remove that is from the database (that should be DAO specific)

Reply:
I don't understand very well the point, because now it doesn't have persistence with the database and in the future, this will change and updated according to the database.